src/apps/frontend/components/ellipsis/ellipsis-menu.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+
+import TaskModal from '../task/task-modal';
+
+const EllipsisMenu: React.FC = () => {
```

Comment:
This is VERY poor abstraction.

You have a generic component inside `components` call ellipsis menu which refers task.

The component name is also not clear